index.js

```diff
@@ -50,19 +53,26 @@ async function run() {
         if (team) {
           const channel = team.slackLabel;
           core.info(`Slack team channel ${channel} found.`);
-          
+
           const slackPostMessageParams = {
-            text: `Le fichier de configuration a été modifié dans la PR *${pullRequest.title}*\n Vérifiez les variables d'environnement d' <${integrationEnvUrl}|intégration>`,
+            text: `Le fichier de configuration a été modifié dans la PR *<${pullRequest.title}|pullRequest.html_url>*\n Vérifiez les variables d'environnement d'<${integrationEnvUrl}|intégration>`,
```

Comment:
Doc GitHub https://docs.github.com/en/rest/commits/commits#list-pull-requests-associated-with-a-commit

Reply:
```suggestion
            text: `Le fichier de configuration a été modifié dans la PR *<${pullRequest.html_url}|${pullRequest.title}>*\n Vérifiez les variables d'environnement d'<${integrationEnvUrl}|intégration>`,
```
(cf https://api.slack.com/reference/surfaces/formatting#linking-urls):
`<http://www.example.com|This message *is* a link>`

Reply:
C'est corrigé